py_trees/src/py_trees/blackboard.py

```diff
@@ -105,26 +108,142 @@ def __str__(self):
         return s
 
 
-class ROSBlackboardMonitor(object):
+class ROSBlackboard(object):
     """
-    Takes in :py:class:`Blackboard <py_trees.blackboard.Blackboard>` class
-    and logs if changes are made
+    Manages :py:class:`Blackboard <py_trees.blackboard.Blackboard>`.
```

Comment:
If this is true, then it should take care of the services that launch/shutdown blackboards as well. These are currently in the ROSBehaviorTree and only manipulate the ros blackboard entity anyway.